src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/TreeNodeDefinition.xml

```diff
@@ -83,6 +83,8 @@
       <Property Name="IsSystemVersioned" ValidFor="Sql2016|Sql2017|AzureV12"/>
 	    <Property Name="TemporalType" ValidFor="Sql2016|Sql2017|AzureV12"/>
       <Property Name="IsExternal" ValidFor="Sql2016|Sql2017|AzureV12|SqlOnDemand"/>
+      <Property Name="IsNode" ValidFor="Sql2017|AzureV12"/>
```

Comment:
Did you look into how this is used? What happens for 2019/2022 servers? Are we correctly adding the property to the default init list for those?